skunkworks_crow/src/main/java/org/odk/share/views/ui/send/fragment/BlankFormsFragment.java

```diff
@@ -156,13 +161,39 @@ private void setEmptyViewVisibility(int len) {
 
     @OnClick(R.id.send_button)
     public void send() {
-        Intent intent = new Intent(getActivity(), SendActivity.class);
+        Intent hotspotIntent = new Intent(getActivity(), HpSenderActivity.class);
+        Intent bluetoothIntent = new Intent(getActivity(), BtSenderActivity.class);
+        setupSendingIntent(hotspotIntent);
```

Comment:
setup should happen after the TRANSFER_OPTION has been selected to prevent extra initialisation of the other intent.

Reply:
Fixed.